.github/workflows/ci.yml

```
@@ -21,12 +21,25 @@ jobs:
       - name: Install dependencies
         run: forge install
 
-      - name: Check backward compatibility
-        run: |
-          forge build --skip test --use solc:0.8.0
-          forge build --skip test --use solc:0.7.6
-          forge build --skip test --use solc:0.7.0
-          forge build --skip test --use solc:0.6.2
+      - name: Check compatibility with 0.8.17 --via-ir
+        if: always()
+        run: forge build --skip test --use solc:0.8.17 --via-ir
```

This is for `Compilation.sol`, right? Do this instead:

```suggestion
        run: forge build --skip test --contracts test/Compilation.sol --use solc:0.8.17 --via-ir
```

---

See above comment